cpp/src/Slice/Parser.cpp

```diff
@@ -3742,12 +3742,10 @@ Slice::InterfaceDef::ids() const
     StringList ids;
     InterfaceList bases = allBases();
     std::transform(bases.begin(), bases.end(), back_inserter(ids), [](const auto& c) { return c->scoped(); });
-    StringList other;
-    other.push_back(scoped());
-    other.push_back("::Ice::Object");
-    other.sort();
-    ids.merge(other);
+    ids.push_back(scoped());
+    ids.push_back("::Ice::Object");
     ids.unique();
+    ids.sort();
```

Comment:
I think you need to sort first.

https://en.cppreference.com/w/cpp/algorithm/unique#:~:text=Eliminates%20all%20except%20the%20first,logical%20end%20of%20the%20range.